packages/owl-bot/test/copy-code.ts

```diff
@@ -0,0 +1,163 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {describe, it} from 'mocha';
+import * as assert from 'assert';
+import {copyDirs} from '../src/copy-code';
+import path from 'path';
+import * as fs from 'fs';
+import glob from 'glob';
+import tmp from 'tmp';
+import {OwlBotYaml} from '../src/config-files';
+
+describe('copyDirs', () => {
+  /**
+   * Creates a sample source tree.
+   */
+  function makeSourceTree(rootDir: string): string {
+    const dirs = [
+      'source',
+      'source/a',
```

Comment:
add an `after` helper that cleans up the directories?

Reply:
They're created in the temp directory, and the OS clears out the temp directory on a regular schedule.